packages/client/mutations/PersistJiraServerSearchQueryMutation.ts

```diff
@@ -0,0 +1,57 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {SimpleMutation} from '../types/relayMutations'
+import {PersistJiraServerSearchQueryMutation as TPersistJiraServerSearchQueryMutation} from '../__generated__/PersistJiraServerSearchQueryMutation.graphql'
+
+graphql`
+  fragment PersistJiraServerSearchQueryMutation_notification on PersistIntegrationSearchQuerySuccess {
+    jiraServerIntegration {
+      searchQueries {
+        id
+        queryString
+        isJQL
+        projectKeyFilters
+      }
+      sharedProviders {
+        id
+      }
+    }
+  }
+`
+
+const mutation = graphql`
+  mutation PersistJiraServerSearchQueryMutation(
+    $teamId: ID!
+    $service: IntegrationProviderServiceEnum!
+    $providerId: Int
+    $jiraServerSearchQuery: JiraServerSearchQueryInput
+  ) {
+    persistIntegrationSearchQuery(
```

Comment:
We still need to wrap the generic mutation in an individual one on a frontend to request the specific integration response above. An issue with the suggestion to improve: #6567